Kernel/Bus/USB/Drivers/USBDriver.h

```diff
@@ -12,6 +12,10 @@
 
 namespace Kernel::USB {
 
+typedef void (*driver_init_function_t)();
```

Comment:
I don't think this has to be so C-y:
```suggestion
using DriverInitFunction = void (*)();
```